src/main/resources/liquibase/module/scripts/v-0.0.1/2023-07-07--00-20-add-queue-item-columns.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+  <changeSet id="2023-07-07--00-20-add-queue-item-columns" author="ncovercash">
```

Comment:
does it make sense to add these columns in the scope of the previously added script for the "dataImportQueueItems" table creation since that script has not been released so the table structure can be fully described there, what do you think?